src/class-methods/class-private-async-generator-method-static-whitespace-syntax-error.case

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2018 Bloomberg LP. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+ /*---
+desc: Syntax error if whitespace after '#' sigil
+template: syntax
+features: [class-methods-private]
+negative:
+  phase: parse
+  type: SyntaxError
```

Comment:
using the templates from `syntax/invalid`, you won't need the negative tag here.